env/export.go

```diff
@@ -35,9 +35,9 @@ var endsWithUnescapedQuoteRegex = regexp.MustCompile("([^\\\\]\"\\z|\\A\"\\z)")
 //     TMP=C:\Users\IEUser\AppData\Local\Temp
 //     USERDOMAIN=IE11WIN10
 //
-func FromExport(body string) *Environment {
+func FromExport(body string) Environment {
 	// Create the environment that we'll load values into
-	env := &Environment{env: make(map[string]string)}
+	env := Environment{}
```

Comment:
Is it a possible problem that we're not `make()`ing the `map` anymore?
I tend to forget the rules around `make()` and map/slice literals.
But testing it quickly… map literal like `Environment{}` or `map[string]string{}` doesn't seem to need `make()` 👌🏼

Reply:
i went down a bit of a rabbithole investigating whether `m := make(map[K]V)` and `m := map[K]V{}` are exactly equivalent and the only answer i can come up with is "probably?". The spec mentions them being "functionally identical", which doesn't mean that they're actually identical ahahha. Regardless, functionally identical is identical enough for me.